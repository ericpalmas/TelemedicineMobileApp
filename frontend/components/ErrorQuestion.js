import React from 'react';
import { NavigationContainer } from '@react-navigation/native';

import {
    StyleSheet,
    Text,
    Button,
    View
} from 'react-native';


const ErrorQuestion = () => {
    return (
        <View style={styles.containerMain}>

            {/* answer area*/}
            <View>
                <Text> Error question</Text>
            </View>
        </View>
    )
}

const styles = StyleSheet.create({
    sectionContainer: {
        marginTop: 32,
        paddingHorizontal: 24,
    },
    sectionTitle: {
        fontSize: 24,
        fontWeight: '600',
    },
    sectionDescription: {
        marginTop: 8,
        fontSize: 18,
        fontWeight: '400',
    },
    highlight: {
        fontWeight: '700',
    },
});

export default ErrorQuestion;