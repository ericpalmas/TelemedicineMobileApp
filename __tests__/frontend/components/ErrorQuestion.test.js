import 'react-native';
import React from 'react';
import ErrorQuestion from '../../../frontend/components/ErrorQuestion';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<ErrorQuestion />).toJSON();
  expect(tree).toMatchSnapshot();
});
