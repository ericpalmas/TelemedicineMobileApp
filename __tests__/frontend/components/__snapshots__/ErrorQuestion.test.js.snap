// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View>
    <Text>
       Error question
    </Text>
  </View>
</View>
`;
